src/libraries/System.Drawing.Common/src/System/Drawing/Graphics.cs

```diff
@@ -1425,7 +1425,7 @@ public void DrawString(string s, Font font, Brush brush, RectangleF layoutRectan
             DrawString(s, font, brush, layoutRectangle, null);
         }
 
-        public void DrawString(string s, Font font, Brush brush, RectangleF layoutRectangle, StringFormat format)
+        public void DrawString(string s, Font font, Brush brush, RectangleF layoutRectangle, StringFormat? format)
```

Comment:
`string s` allowing null, not sure if intended though.

And please make  `StringFormat? format` nullable for all overloads